ert_shared/main.py

```diff
@@ -458,5 +458,19 @@ def main():
         sys.exit(str(err))
     except:
         logger.exception("ert crashed unexpectedly")
-        sys.exit("ert crashed unexpectedly")
+
+        logfiles = set()  # Use set to avoid duplicates...
+        all_loggers = [  # Call getLogger() to ensure the logger is initialised
+            logging.getLogger(name) for name in logging.root.manager.loggerDict
+        ]
+        # List-comprehension above does not include root-logger, add it manually
+        for lgger in [logging.getLogger()] + all_loggers:
+            for handler in lgger.handlers:
+                if isinstance(handler, logging.FileHandler):
+                    logfiles.add(handler.baseFilename)
+
+        msg = "ERT crashed unexpectedly. See logfiles for details:"
+        msg += "\n   " + "\n   ".join(logfiles)
+
+        sys.exit(msg)
```

Comment:
Could also replace this with: 

```python
except:
    error = sys.exc_info()[1]
    print(f"ERT crashed unexpectedly with error: {error}")
```

Reply:
Or perhaps split it into:
```python
except Exception as err:
    print(f"ERT crashed unexpectedly with: {str(error)}")
except BaseException as err:
    some custom handling here
```

Reply:
I agree. Perhaps even
```
except Exception as ex:
    print(f"ERT crashed unexpectedly with error: {ex}")
```
..  or?

Reply:
Hmm...  missed your second suggestion. It's a good point to use BaseException to ensure we grab everything unexpected... I don't see, however, why we would distinguish between an unexpected `Exception` and an unexpected `BaseException`  😄 

Updated code produces the following if I downgrade `ert-storage` to e.g. `0.3.1`
```
MacBook-Air:poly_example bjarne$  ert ensemble_experiment poly.ert
ERT crashed unexpectedly with "The 'ert-storage>=0.3.4' distribution was not found and is required by ert".
See logfiles for details:
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/ert-log.txt
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/ee-log.txt
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/api-log.txt
MacBook-Air:poly_example bjarne$ 
```

Reply:
> I don't see, however, why we would distinguish between an unexpected Exception and an unexpected BaseException 😄

`BaseException` would for example include `KeyboardInterrupt` and `SystemExit`, which we might want to treat differently then for example `ValueError`, was my rationale at least.

Reply:
Yes, but I still don't see what we can / wish to do differently. 😄   ( I'm open to suggestions!)